.github/workflows/docker-base-image.yml

```diff
@@ -1,55 +1,58 @@
 name: Docker Base Image CI
 
 on:
-  workflow_dispatch:
-    inputs:
-      node_version:
-        description: 'Node.js version to build this image with.'
-        type: choice
-        required: true
-        default: '20'
-        options:
-          - '20'
-          - '22'
-          - '24'
+  push:
+    branches:
+      - master
+    paths:
+      - 'docker/images/n8n-base/Dockerfile'
+  pull_request:
+    paths:
+      - 'docker/images/n8n-base/Dockerfile'
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: ['20', '22', '24']
```

Comment:
Could we drop 20?

Reply:
We only consume the 22 version internally, so I'm actually not sure why we push both 20/24. Do we have a "supported" list?

Reply:
We still run the unit tests against 20
        node-version: [20.x, 22.x, 24.x]
So I'll keep this in the PR for now, and we can drop it in a different PR if needed?

Reply:
I don't think we have it explicitly written down anywhere. I think we support node 20 since it's still on maintenance.